SConstruct

```diff
@@ -880,18 +895,31 @@ if env['system_fmt'] in ('n', 'default'):
                          'Try manually checking out the submodule with:\n\n'
                          '    git submodule update --init --recursive ext/fmt\n')
 
+fmt_include = '<fmt/format.h>' if env['system_fmt'] else '"../ext/fmt/include/fmt/format.h"'
+fmt_version_source = get_expression_value([fmt_include], 'FMT_VERSION')
+retcode, fmt_lib_version = conf.TryRun(fmt_version_source, '.cpp')
+try:
+    fmt_lib_version = divmod(float(fmt_lib_version.strip()), 10000)
+    (fmt_maj, (fmt_min, fmt_pat)) = fmt_lib_version[0], divmod(fmt_lib_version[1], 100)
+    env['FMT_VERSION'] = '{major:.0f}.{minor:.0f}.{patch:.0f}'.format(major=fmt_maj, minor=fmt_min, patch=fmt_pat)
+    print('INFO: Found fmt version {}'.format(env['FMT_VERSION']))
+except ValueError:
+    print('INFO: Could not find version of fmt')
+
 # Check for googletest and checkout submodule if needed
 if env['system_googletest'] in ('y', 'default'):
     has_gtest = conf.CheckCXXHeader('gtest/gtest.h', '""')
     has_gmock = conf.CheckCXXHeader('gmock/gmock.h', '""')
     if has_gtest and has_gmock:
         env['system_googletest'] = True
+        print("""INFO: Using system installation of googletest library.""")
     elif env['system_googletest'] == 'y':
         config_error('Expected system installation of Googletest-1.8.0, but it '
                      'could not be found.')
 
 if env['system_googletest'] in ('n', 'default'):
     env['system_googletest'] = False
+    print("""INFO: Using private installation of googletest library.""")
```

Comment:
Can we not do this right now? I'm anticipating merge conflicts with #535 if there are any changes to the googletest options.